EDAM_dev.owl

```diff
@@ -53380,6 +53380,9 @@ ows re-sequencing of complete genomes of any given organism with high resolution
 
     <owl:Class rdf:about="http://edamontology.org/topic_0199">
         <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_3321"/>
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_3053"/>
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_3070"/>
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_0003"/>
```

Comment:
These 3 super-classes that you added are _Genetics_, _Biology_, and _Topic_. These are all implied already via the [Molecular genetics](http://edamontology.org/topic_3321) "parent" class, so we don't need to (and don't want to) specify them here again.

Could you please **instead** of these 3 add as a super-class the **_Genomics_** concept? You'll find its URI when you search in https://bioportal.bioontology.org/ontologies/EDAM/?p=classes. The "Jump to" is the search bar there (a bit non-intuitive). Now that you know how to edit `EDAM_dev.owl`, this should be quick to fix for you

The problem with your Git branch: Please create the new branch from the `main` branch in your repository. Otherwise it's impossible to merge. (In case you need more help with that, the [Git hands-on tutorial](https://coderefinery.github.io/git-intro/) is very useful and it should be learner-friendly if you go through it yourself.)

Reply:
One more trick: in case you are using Firefox as your web browser, you may want to add the **EDAM Popovers** extension. It helps that when you select a piece of text including an EDAM URI (ID) in a website open in Firefox, it will show details about that concept.

Reply:
Ok @matuskalas

Reply:
Awesoooomeee 😸